fighthealthinsurance/temporal_client.py

```
@@ -0,0 +1,131 @@
+def dispatch_fax_send_blocking(hashed_email: str, fax_uuid: str) -> Optional[bool]:
+    """Run a fax send via Temporal and block until it finishes.
+
+    Returns the send result (True/False) when handled by Temporal, or None when
+    Temporal is disabled or the dispatch failed -- in which case the caller
+    should fall back to the blocking Ray path.
+    """
+    if not getattr(settings, "TEMPORAL_ENABLED", False):
+        return None
+    try:
+        return async_to_sync(execute_send_fax_workflow)(hashed_email, str(fax_uuid))
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)